prime-router/metadata/hl7_mapping/datatypes/HumanName/XPN.yml

```diff
@@ -0,0 +1,77 @@
+elements:
+
+  - name: xpn-family-name
+    value: [ '%resource.family' ]
+    hl7Spec: [ '%{hl7xpnField}-1-1' ]
+
+  - name: xpn-humanname-own-prefix
+    value: [ '%resource.family.extension(%`ext-humanname-own-prefix`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-2' ]
+
+  - name: xpn-humanname-own-name
+    value: [ '%resource.family.extension(%`ext-humanname-own-name`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-3' ]
+
+  - name: xpn-humanname-partner-prefix
+    value: [ '%resource.family.extension(%`ext-humanname-partner-prefix`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-4' ]
+
+  - name: xpn-humanname-partner-name
+    value: [ '%resource.family.extension(%`ext-humanname-partner-name`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-5' ]
```

Comment:
Thoughts on extracting these 5 into a `HumanName/FN.yml`? I guess it would only be needed here and in XCN, but still could be worthwhile.